Api/repository/entities/Budget.cs

```diff
@@ -0,0 +1,9 @@
+public class Budget
```

Comment:
The scope of your US wasn't to create entities, and you are not following the UML

Reply:
https://docs.google.com/document/d/1rTQ4vGkdmmXnmc9xuSMiI3QpoGo1o9638-eMgYkCOi8/edit

Sorry, take reference in this file now replace it with data from uml.